src/noisepy/seis/main.py

```diff
@@ -59,8 +59,10 @@ def get_channel_filter(sta_list: List[str]) -> Callable[[Channel], bool]:
 
 
 def create_raw_store(raw_dir: str, sta_list: List[str], xml_path: str):
+    fs = get_filesystem(raw_dir)
+
     def count(pat):
-        return len(glob(os.path.join(raw_dir, pat)))
+        return len(fs.glob(os.path.join(raw_dir, pat)))
```

Comment:
Does this one matter if it's not using your `fs_join`?

Reply:
Good catch! it does matter, I'll change it ...